### PR TITLE
[GHA] Update MO deps

### PR DESCRIPTION
**Pull request by rkazants**
**Details:** Update MO deps

**Ticket:** 129336

### CODE REVIEW

File: .github/components.yml [hunk 1]
```diff
@@ -199,6 +199,11 @@ IE_Tests:
     - IR_FE
 
 MO:
+  revalidate:
+    - PyTorch_FE
+    - TF_FE
+    - ONNX_FE
+    - PDPD_FE
```
Comment (ilya-lavrenov) [inline .github/components.yml:206]: do  you mean that `convert_model` must revalidate frontends?
how PDPD is affected by changes in MO / OVC?

Comment (rkazants) [reply .github/components.yml:206]: MO is responsible for frontend configuration. It invoked FE API for this configuration. So if we change something in this common logic incorrectly, we can break conversion of some FW models, for example.

Comment (rkazants) [reply .github/components.yml:206]: If PDPD FE does not have e2e tests in precommit, I can remove it.

Comment (ilya-lavrenov) [reply .github/components.yml:206]: MO is built on top of frontends => frontends does not depend on MO, does they?

Comment (ilya-lavrenov) [reply .github/components.yml:206]: ok, let's keep only those FEs which rely on OVC / MO a lot

Comment (rkazants) [reply .github/components.yml:206]: > MO is built on top of frontends => frontends does not depend on MO, does they?

This dependency is unfortunately created due to these integration tests.

Comment (rkazants) [inline .github/components.yml:206]:
```suggestion
    - PyTorch_FE
    - TF_FE
```

Comment (rkazants) [reply .github/components.yml]: PR and TF are left.

**Review by mvafin — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**